stage1/init/init.mk

```diff
@@ -1 +1,2 @@
+BGB_GO_FLAGS := $(strip -tags "$(KVM_HV_TAG)")
```

Comment:
Maybe extend `RKT_TAGS` variable in `Makefile.in` and just use it here as `BGB_GO_FLAGS := $(strip $(RKT_TAGS))`?